.github/workflows/continuous-deployment.yaml

```diff
@@ -52,13 +54,13 @@ jobs:
       - name: 📦 Install dependencies
         run: pnpm install
 
-      - name: 🚀 Build 
+      - name: 🚀 Build
         run: pnpm build
 
       - name: 🚀 Build internal preview
         if: ${{ github.event.inputs.channel == 'preview' }}
-        run: (cd apps/${{ github.event.inputs.app }} && eas build --non-interactive --platform ${{ github.event.inputs.platform }} --profile preview --no-wait)
+        run: eas build --non-interactive --platform ${{ github.event.inputs.platform }} --profile ${{ env.APP_PROFILE }}-preview --no-wait
```

Comment:
can we skip the cd? I think we still need to run it from the `apps/easypid` folder right?